DockerSettings.php

```diff
@@ -402,72 +404,10 @@
 $tmpProxy = getenv( 'MW_PROXY_SERVERS' );
 if ( $tmpProxy ) {
 	# https://www.mediawiki.org/wiki/Manual:Varnish_caching
-	$wgUseSquid = true;
-	$wgSquidServers = explode( ',', $tmpProxy );
+	$wgUseCdn = true;
+	$wgCdnServers = explode( ',', $tmpProxy );
 	$wgUsePrivateIPs = true;
-	$wgHooks['IsTrustedProxy'][] = function( $ip, &$trusted ) {
```

Comment:
I removed the IsTrustedProxy hook because $wgCdnServersNoPurge does the same what the hook did.